resources/hiveclients/python/hiveserver2-client.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# imports
+import sys
+import pyhs2
+from pyhs2.error import Pyhs2Exception
+
+# get the input parameters
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c